sonic_platform_base/sonic_xcvr/xcvr_api_factory.py

```diff
@@ -90,10 +96,17 @@ def create_xcvr_api(self):
             api = Sff8636Api(xcvr_eeprom)
         # QSFP+
         elif id == 0x0D:
-            codes = Sff8436Codes
-            mem_map = Sff8436MemMap(codes)
-            xcvr_eeprom = XcvrEeprom(self.reader, self.writer, mem_map)
-            api = Sff8436Api(xcvr_eeprom)
+            revision_compliance = self._get_revision_compliance()
```

Comment:
@stephenxs can you quote the SPEC screenshot for this compliance logic?

Reply:
shared them with you offline.

Reply:
@stephenxs thanks